root/collection/CollectionIndex.js

```diff
@@ -190,7 +190,7 @@ const CollectionIndex = (props: Props) => {
         ) : null}
       </div>
       <div className="">
-        {collection.collaborators && collection.collaborators.length ? (
+        {collection.collaborators.length ? (
```

Comment:
For example, here the check looked unnecessary, since we always output `collaborators` in `TO_JSON` (and it calls `collection.collaborators.some` above without the extra check, so it would've been already broken otherwise).